cli/src/hellaswag_gpt.ts

```diff
@@ -0,0 +1,55 @@
+import '@tensorflow/tfjs-node';
+import { loadHellaSwag } from '@epfml/discojs-node';
+import { models } from '@epfml/discojs';
+import { AutoTokenizer, PreTrainedTokenizer } from '@xenova/transformers';
+import fs from 'fs';
+import path from 'node:path';
+
+const hellaswag_path = path.join('..', 'datasets', 'hellaswag_val.jsonl');
+const logFile = path.join('..', 'datasets', 'LogFile_hellaswag.txt');
+const logLines: string[] = [];
+
+function log(message: string) {
+    console.log(message);
+    logLines.push(message);
+}
+
+const hellaswagDataset: models.HellaSwagDataset = await loadHellaSwag(hellaswag_path)

```

Comment:
hellaswag should be downloaded here rather than assuming that it has already been downloaded (or decide to not download it at all and let `loadHellaSwag` fetch it with `fs.createReadStream(HELLASWAG_URL)`)